src/lib/io/sol-iio.c

```diff
@@ -36,17 +36,24 @@
 #include <sol-util-file.h>
 #include <sol-vector.h>
 
+#include "sol-list.h"
```

Comment:
May I ask you why do you favor `sol-list` over `sol-vector` or `sol-ptr-vector`?